orttraining/orttraining/training_ops/cuda/nn/dropout.h

```diff
@@ -21,5 +21,22 @@ class DropoutGrad final : public CudaKernel {
   const float default_ratio_;
 };
 
+template <typename T1, typename T2>
+class BiasDropout final : public CudaKernel {
+ public:
+  BiasDropout(const OpKernelInfo& info) : CudaKernel(info), default_ratio_(0.5) {
+    int64_t seed = 0;
+    if (info.GetAttr<int64_t>("seed", &seed).IsOK()) {
+      generator_ = onnxruntime::make_unique<PhiloxGenerator>(static_cast<uint64_t>(seed));
+    }
+  }
+
+  Status ComputeInternal(OpKernelContext* context) const override;
+
+ private:
+  mutable std::unique_ptr<PhiloxGenerator> generator_;
+  const float default_ratio_;
```

Comment:
>default_ratio_ [](start = 14, length = 14)

looks like this can be static, maybe even a constant shared between BiasDropout and DropoutGrad

Reply:
made it static constexpr 

---
In reply to: [441078925](https://github.com/microsoft/onnxruntime/pull/4167#discussion_r441078925) [](ancestors = 441078925)